input.css

```diff
@@ -30,6 +30,10 @@
     @apply block w-full mb-6 bg-gray-50 border border-gray-300 rounded-lg focus:ring-blue-500 focus:border-blue-500;
   }
 
+  .casestudy-sel {
```

Comment:
The down arrow for select elements are difficult to style. This style change replaces the native arrow with an SVG image so that space could be added to the right side. The change prevents the arrow from appearing cramped near the border.